server.js

```diff
@@ -1,27 +1,92 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
+import express from 'express'
+import cors from 'cors'
+import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
+const mongoUrl =
+  process.env.MONGO_URL || 'mongodb://localhost/project-happyhappyThoughts'
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
+const port = process.env.PORT || 8080
+const app = express()
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
+
+app.use(cors())
+app.use(express.json())
+
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: 'Service not online' })
+  }
+})
+
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    minlength: 5,
+    maxlength: 140,
+    required: true,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createAt: {
+    type: String,
+    default: () =>
```

Comment:
When I test your app, the time was recorded 2 hours ahead of local (Swedish) time. In my submission, I create new Date object and assigned it to createdAt field when client send POST request. If you really wish to be super accurate, maybe you can check out my solution. But I think this can be easily solved in client-side (just adding two hours when displaying the date in browser)!